Source/WebGPU/WGSL/WGSL.h

```diff
@@ -47,15 +47,15 @@ class ShaderModule;
 struct SuccessfulCheck {
     SuccessfulCheck() = delete;
     SuccessfulCheck(SuccessfulCheck&&);
-    SuccessfulCheck(Vector<Warning>&&, UniqueRef<ShaderModule>&&);
+    SuccessfulCheck(WTF::Vector<Warning>&&, UniqueRef<ShaderModule>&&);
```

Comment:
Can we rejigger our includes to make this not have to be changed?

Reply:
Yeah, that was 100% a mistake, I'll upload a patch to move the types into their own namespace.

Reply:
Preferably `WGSL::Type::Vector`

Reply:
Yes, I moved it in #9976. Only I made it `Types`, because I wanted `Type` to remain the base class that lives in the top-level `WGSL` namespace.